pyproject.toml

```diff
@@ -13,7 +13,7 @@ authors = [
 description = "Project to generate recipes for conda packages"
 readme = "README.md"
 keywords = ["conda"]
-license = {text = "MIT"}
+license = {text = "Apache-2.0"}
```

Comment:
Thank you! I forgot to update this part! grayskull used to be MIT, but I changed it to Apache-2.0